tox.ini

```diff
@@ -27,6 +27,7 @@ description =
 setenv =
     MPLBACKEND = agg
     SUNPY_SAMPLEDIR = {env:SUNPY_SAMPLEDIR:{toxinidir}/.tox/{envname}/sample_data/}
+    ci: SUNPY_SAMPLEDIR = {toxinidir}/docs/sample_data/
```

Comment:
That is where the sample data gets downloaded to in the CI?!

Reply:
Why can't we just cache the default directory? or set this env var inside the github actions workflow?

Reply:
In tox the default (from the previous line) is e.g. `.tox/build_docs/sample_data/` or `.tox/build_docs-gallery/sample_data/` so we can't cache and restore easily as they are different directories. So, yes, *in the docs builds only* I force it to be `docs/sample_data/` instead. Unless there would be a better common directory?

We can't pass env vars from a workflow to a reusable workflow. https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations

Reply:
Why don't we change the line above so all the envs use the same sample data in `.tox/sample_data` then we can restore the same cache for the online builds later etc?

Reply:
The existing files shouldn't ever change.

Reply:
Okay I'll make `.tox/sample_data` the default.